README.md

```diff
@@ -405,9 +409,74 @@ Expected output:
 (20,28,null,placename_settlement,"Warszawie","Warszawie")
 ```
 
+
+### PolEval 2019 Task 1: Recognition and normalisation of temporal expressions
```

Comment:
Czy na pewno jest podlinkowany właściwy model? Nazwa paczki z modelem i output wskazują, że jest to model dla sytuacji, a nie timex-ów.

Reply:
Jest to cała paczka wszystkich modeli, które były w Liner2.5 na DSpace. Do podzielenia na mniejsze modele i wrzucenie ich osobno na DSpace.